### PR TITLE
[New Feature] Touchpad gestures

### PR DESCRIPTION
**Pull request by mahaveergurjar**
# Pull Request

## Description

Add touchpad gestures for laptop users 
user can customise the gesture using gesture app or changing in libinput-gestures.conf file 

## Type of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [x] I have added necessary comments/documentation to my code.
- [x] I have added tests to cover my changes.
- [x] I have tested my code locally and it works as expected.
- [x] All new and existing tests passed.

## Screenshots

240316_23h51m51s_screenshot
240316_23h52m02s_screenshot

### CODE REVIEW

File: Configs/.config/hypr/userprefs.t2 [hunk 1]
```diff
@@ -27,6 +27,7 @@ bind = $mainMod ALT, XF86MonBrightnessUp, exec, hyprshade off # disable blue lig
 
 # exec-once = swayidle -w timeout 600 'swaylock' timeout 900 'hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on' # lock after 10 mins, turn off monitor after 15 mins // install swayidle
 exec-once = swayidle -w timeout 1200 'swaylock; hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on' timeout 1800 'systemctl suspend' # lock & turn off monitor after 20 mins, suspend after 30 mins // install swayidle
+exec-once = command -v libinput-gestures && libinput-gestures
```
Comment (kRHYME7) [inline Configs/.config/hypr/userprefs.t2:30]: ``` exec-once = libinput-gestures ```

Should be enough as users will pick this up if they want to use this.

Comment (mahaveergurjar) [reply Configs/.config/hypr/userprefs.t2:30]: ok

**Comment by prasanthrangan** [comment]
Thank you, I don't have a touchpad to test... so just let me know when its ready to merge.

**Comment by mahaveergurjar** [comment]
OK when @kRHYME7 tests this then this will be ready for merge

**Review by kRHYME7 — APPROVED** [review]
Looks Okay

**Review by prasanthrangan — APPROVED** [review]
alright... thanks for testing it out!

**Comment by mugiwara999** [comment]
Hey can you tell me how to make this work , I have run the install script but the gestures you mentioned doesn't work

**Comment by kRHYME7** [comment]
@mugiwara999 

 ``` exec-once = libinput-gestures ```

You should add this to your userprefs.conf first then relogin/ reboot. the libinput-gestures daemon should be running

**Comment by mugiwara999** [comment]
@kRHYME7 I did it but it's not working

**Pull request merged**